utils/mailer.js

```diff
@@ -1,19 +1,17 @@
 const nodemailer = require('nodemailer')
 const { google } = require('googleapis')
 const logger = require('~/logger/logger')
-const requiredCredentials = require('~/consts/gmailAuth')
+const {
+  gmailCredentials: { user, clientId, clientSecret, refreshToken, redirectUri }
+} = require('~/configs/config')
 
 const OAuth2 = google.auth.OAuth2
 
 const getAccessToken = async () => {
   try {
-    const oAuth2Client = new OAuth2(
-      requiredCredentials.clientId,
-      requiredCredentials.clientSecret,
-      requiredCredentials.redirectUri
-    )
+    const oAuth2Client = new OAuth2(clientId, clientSecret, redirectUri)
 
-    oAuth2Client.setCredentials({ refresh_token: requiredCredentials.refreshToken })
+    oAuth2Client.setCredentials({ refresh_token: refreshToken })
```

Comment:
you can do oAuth2Client.setCredentials({ refresh_token })

Reply:
sure, i could name it your way if it was the only place where i need to use this refresh token, but i also have refresherToken field in transport auth